### PR TITLE
Tempus 995 freemium account

### PR DESCRIPTION
**Pull request by NandiniMGawade**
Thank you for submitting a contribution to Tempus.

In order to streamline the review of the contribution please
ensure the following steps have been taken:

### For all changes:
- [x] Is there a Waffle ticket associated with this PR? Is it referenced 
     in the commit message?

- [x] Does your PR title start with Tempus-XXXX where XXXX is the waffle number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [x] Has your PR been rebased against the latest commit within the target branch (typically dev)?

- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed via mvn clean install at the root Tempus folder?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [ ] Have you added documentation for any UI or API related changes to the /docs folder

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: ui/src/app/signup/activation-link.controller.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+
+    function reSendLink() {
+        signUpService.resentEmailLink($stateParams.email).then(
+             function success() {
+                 vm.buttonDisabled = true;
+                toast.showSuccess($translate.instant('signup.emailSuccess'));
+                $timeout(function(){
+                        vm.buttonDisabled = false;
+                    },50000);
```
Comment (Shobhit2884) [inline ui/src/app/signup/activation-link.controller.js]: why timeout for 5 minutes?

**Comment by Shobhit2884** [comment]
UI Part LGTM

File: application/src/main/java/com/hashmapinc/server/controller/UserController.java [hunk 2]
```diff
@@ -132,6 +138,49 @@ public User saveUser(@RequestBody User user,
         }
     }
 
+    @PostMapping(value = "/noauth/user")
+    @ResponseBody
+    public User saveTrialUser(@RequestBody User user,
+                         HttpServletRequest request) throws TempusException {
+        Tenant savedTenant = null;
+        try {
+            if (user.getAuthority() == Authority.TENANT_ADMIN) {
```
Comment (Himanshu-it) [inline application/src/main/java/com/hashmapinc/server/controller/UserController.java]: as discussed do not sent user authority from ui, set it in the backend.

**Review by Himanshu-it — APPROVED** [review]

**Review by Shobhit2884 — APPROVED** [review]
UI Part with Latest changes are Ok

**Pull request merged**